content/blog/2017-03-07-engineering-leads-and-mothers.md

```diff
@@ -0,0 +1,40 @@
++++
+date = "2017-03-06T13:14:19+01:00"
+title = "What new engineering leads and new parents have in common"
+
++++
+
+Two of the biggest changes in my life over the last year in work and personal life were moving from being an engineering more into a management/leadership role and having a kid 5 months ago with my girlfriend.
+
```

Comment:
I'd substitute `child` for `kid` in this post. While `kid` is definitely correct, it has a really informal (and at times, even negative) tone, which doesn't match the tone of the article.